src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -178,10 +177,10 @@
       {{ paidInFullCCJCancelledClaimantClaimDetails(claim) }}
     {% elif state.status === ClaimStatus.PAID_IN_FULL_CCJ_SATISFIED %}
       {{ paidInFullCCJSatisfiedClaimantClaimDetails(claim) }}
-    {% elif (state.status === ClaimStatus.NO_RESPONSE or state.status === ClaimStatus.ELIGIBLE_FOR_CCJ) and claim.helpWithFeesNumber and claim.state === 'HWF_APPLICATION_PENDING' and claim.lastEventTriggeredForHwfCase === 'RecalculateInterest' %}
+    {# {% elif (state.status === ClaimStatus.NO_RESPONSE or state.status === ClaimStatus.ELIGIBLE_FOR_CCJ) and claim.helpWithFeesNumber and claim.state === 'HWF_APPLICATION_PENDING' and claim.lastEventTriggeredForHwfCase === 'RecalculateInterest' %}
```

Comment:
If this code is no longer required (removed as part of your story), please remove it instead of commenting it.